sdk/keyvault/keyvault-common/package.json

```diff
@@ -49,6 +49,7 @@
     "tslib": "^2.0.0"
   },
   "devDependencies": {
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
```

Comment:
Although I added the plugin as a dep here, for some reason `rushx lint` fails with the following message:
```
Oops! Something went wrong! :(

ESLint: 7.3.1

ESLint couldn't find the plugin "@azure/eslint-plugin-azure-sdk".

(The package "@azure/eslint-plugin-azure-sdk" was not found when loaded as a Node module from the directory "/home/dalmahal/repo5/sdk".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install @azure/eslint-plugin-azure-sdk@latest --save-dev

The plugin "@azure/eslint-plugin-azure-sdk" was referenced from the config file in "../../.eslintrc.json".

If you still can't figure out the problem, please stop by https://eslint.org/chat to chat with the team.
```

Reply:
I needed to add `eslint` with the correct semvar to deps. Fixed.